packages/shared/components/popups/Version.svelte

```diff
@@ -25,10 +29,13 @@
 
     onMount(async () => {
         // @ts-ignore: This value is replaced by Webpack DefinePlugin
-        /* eslint-disable no-undef */
-        if (!devMode) {
+        if (devMode) {
             await getVersionDetails()
-            updateCheck()
+            if (get(stage) === Stage.PROD) {
+                updateCheck()
+            } else {
+                isPreRelease = true
+            }
         }
```

Comment:
This doesn't work, because a pre-release version is anything that is not prod. Where this check is on devmode only.

Reply:
Whoops, I changed that for testing purposes and then forgot to change it back